parm/land/prep/prep_run.yaml

```diff
@@ -0,0 +1,31 @@
+    - [$(DATA)/bkg/{{ current_cycle | to_YMD }}.{{ current_cycle.strftime('%H') }}0000.sfc_data.tile1.nc, $(DATA)/bkg/mem002/RESTART/{{ current_cycle | to_YMD }}.{{ current_cycle.strftime('%H') }}0000.sfc_data.tile1.nc]
+    - [$(DATA)/bkg/{{ current_cycle | to_YMD }}.{{ current_cycle.strftime('%H') }}0000.sfc_data.tile2.nc, $(DATA)/bkg/mem002/RESTART/{{ current_cycle | to_YMD }}.{{ current_cycle.strftime('%H') }}0000.sfc_data.tile2.nc]
+    - [$(DATA)/bkg/{{ current_cycle | to_YMD }}.{{ current_cycle.strftime('%H') }}0000.sfc_data.tile3.nc, $(DATA)/bkg/mem002/RESTART/{{ current_cycle | to_YMD }}.{{ current_cycle.strftime('%H') }}0000.sfc_data.tile3.nc]
+    - [$(DATA)/bkg/{{ current_cycle | to_YMD }}.{{ current_cycle.strftime('%H') }}0000.sfc_data.tile4.nc, $(DATA)/bkg/mem002/RESTART/{{ current_cycle | to_YMD }}.{{ current_cycle.strftime('%H') }}0000.sfc_data.tile4.nc]
+    - [$(DATA)/bkg/{{ current_cycle | to_YMD }}.{{ current_cycle.strftime('%H') }}0000.sfc_data.tile5.nc, $(DATA)/bkg/mem002/RESTART/{{ current_cycle | to_YMD }}.{{ current_cycle.strftime('%H') }}0000.sfc_data.tile5.nc]
+    - [$(DATA)/bkg/{{ current_cycle | to_YMD }}.{{ current_cycle.strftime('%H') }}0000.sfc_data.tile6.nc, $(DATA)/bkg/mem002/RESTART/{{ current_cycle | to_YMD }}.{{ current_cycle.strftime('%H') }}0000.sfc_data.tile6.nc]
+increment:
+  copy:
+    - [$(DATA)/anl/landinc.{{ current_cycle | to_YMD }}.{{ current_cycle.strftime('%H') }}0000.coupler.res, $(DATA)/anl/{{ current_cycle | to_YMD }}.{{ current_cycle.strftime('%H') }}0000.xainc.coupler.res]
+    - [$(DATA)/anl/landinc.{{ current_cycle | to_YMD }}.{{ current_cycle.strftime('%H') }}0000.sfc_data.tile1.nc, $(DATA)/anl/{{ current_cycle | to_YMD }}.{{ current_cycle.strftime('%H') }}0000.xainc.sfc_data.tile1.nc]
+    - [$(DATA)/anl/landinc.{{ current_cycle | to_YMD }}.{{ current_cycle.strftime('%H') }}0000.sfc_data.tile2.nc, $(DATA)/anl/{{ current_cycle | to_YMD }}.{{ current_cycle.strftime('%H') }}0000.xainc.sfc_data.tile2.nc]
+    - [$(DATA)/anl/landinc.{{ current_cycle | to_YMD }}.{{ current_cycle.strftime('%H') }}0000.sfc_data.tile3.nc, $(DATA)/anl/{{ current_cycle | to_YMD }}.{{ current_cycle.strftime('%H') }}0000.xainc.sfc_data.tile3.nc]
+    - [$(DATA)/anl/landinc.{{ current_cycle | to_YMD }}.{{ current_cycle.strftime('%H') }}0000.sfc_data.tile4.nc, $(DATA)/anl/{{ current_cycle | to_YMD }}.{{ current_cycle.strftime('%H') }}0000.xainc.sfc_data.tile4.nc]
+    - [$(DATA)/anl/landinc.{{ current_cycle | to_YMD }}.{{ current_cycle.strftime('%H') }}0000.sfc_data.tile5.nc, $(DATA)/anl/{{ current_cycle | to_YMD }}.{{ current_cycle.strftime('%H') }}0000.xainc.sfc_data.tile5.nc]
+    - [$(DATA)/anl/landinc.{{ current_cycle | to_YMD }}.{{ current_cycle.strftime('%H') }}0000.sfc_data.tile6.nc, $(DATA)/anl/{{ current_cycle | to_YMD }}.{{ current_cycle.strftime('%H') }}0000.xainc.sfc_data.tile6.nc]
```

Comment:
Why can't JEDI write it in the format that is expected?
See changes to `letfkoi.yaml` in the branch `feature/land_analysis`.

Reply:
This is a good question, I thought we discussed this previously @jiaruidong2017 to modify the JEDI YAML to write out the expected filenames

Reply:
It doesn't work? I thought for sure `prefix` was optional...

Reply:
In that case, I suggest set `prefix: landinc` and remove `xainc` from the destination filename.
This will be consistent w/ `aeroinc` in aerosol DA update.
An update may be required in `apply_incr.exe` to take this updated filename.
In general, minimize file movements and have a plan for how the data "flows" through the system.

Reply:
@CoryMartin-NOAA I will try it again, and let you know then.

Reply:
I made an experiment by removing the `prefix:` from the letkfoi.yaml as below:

```
output increment:
  datapath: $(DATA)/anl
  filetype: fms restart
  filename_sfcd: '{{ current_cycle | to_fv3time }}.xainc.sfc_data.nc'
  filename_cplr: '{{ current_cycle | to_fv3time }}.xainc.coupler.res'
  state variables: [snwdph,vtype,slmsk]
```

The output from JEDI run is as below:
```
20210323.060000.20210323.060000.xainc.coupler.res
20210323.060000.20210323.060000.xainc.sfc_data.tile1.nc
20210323.060000.20210323.060000.xainc.sfc_data.tile2.nc
20210323.060000.20210323.060000.xainc.sfc_data.tile3.nc
20210323.060000.20210323.060000.xainc.sfc_data.tile4.nc
20210323.060000.20210323.060000.xainc.sfc_data.tile5.nc
20210323.060000.20210323.060000.xainc.sfc_data.tile6.nc
```
The output files have doubled times of `20210323.060000`. 

If I kept the option `prefix:` in the `letkfoi.yaml` as below: 
```
output increment:
  datapath: $(DATA)/anl
  prefix:
  filetype: fms restart
  filename_sfcd: '{{ current_cycle | to_fv3time }}.xainc.sfc_data.nc'
  filename_cplr: '{{ current_cycle | to_fv3time }}.xainc.coupler.res'
  state variables: [snwdph,vtype,slmsk]
```
The output is the same as above with double times. 

Do you have any suggestions? Thanks.

Reply:
Well that is weird... seems like a bug. But can you not just write:
```
  filename_sfcd: 'xainc.sfc_data.nc'
  filename_cplr: 'xainc.coupler.res'
```
?

Reply:
@CoryMartin-NOAA Do you want me just try: 
```
  filename_sfcd: 'xainc.sfc_data.nc'
  filename_cplr: 'xainc.coupler.res'
```

Reply:
yes

Reply:
It is weird, because it produced the same output as previous.

Reply:
Please ignore my previous answers. The above settings could produce the correct output as below: 

```
20210323.060000.xainc.coupler.res
20210323.060000.xainc.sfc_data.tile1.nc
 20210323.060000.xainc.sfc_data.tile2.nc
20210323.060000.xainc.sfc_data.tile3.nc
20210323.060000.xainc.sfc_data.tile4.nc
20210323.060000.xainc.sfc_data.tile5.nc
20210323.060000.xainc.sfc_data.tile6.nc
```
It seems if prefix is not defined, the time tag will be added at the beginning by default. Therefore, we can use this settings to avoid copying/renaming files.